src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, onTabSelected, activeTabId }) => {
+  const isActiveTabValid = tabs.some(tab => tab.id === activeTabId);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab, index) => {
+            return (
+              <li
+                className={`${(!isActiveTabValid && index === 0) || activeTabId === tab.id ? 'is-active' : ''}`}
```

Comment:
Scary part of the code :)

Reply:
> Scary part of the code :)

Yeah agree... Looks like some kind of mental illness include depression, anxiety disorders and schizophrenia🫠